sdk/identity/identity/README.md

```diff
@@ -233,6 +240,10 @@ setLogLevel("info");
 
 ### Read the documentation
 
+The client libraries listed [here][releases] support authenticating with `TokenCredential` and the Azure Identity library. You can learn more about their usage and find additional documentation along with samples in the links mentioned [here][releases]
```

Comment:
Are we sure we want to link to https://azure.github.io/azure-sdk/releases/latest/js.html? Once we land there, we have a tab for "management libraries" which are all Track 1 today and dont support TokenCredential. We are working on making this work in the next month or so, but there is still another tab "All" which is a list of ALLLL JS packages related to azure

Reply:
Any thoughts on this?

Reply:
It's a good point. Do we have a definitive list of only Track 2 libraries that we can link to instead? 

Alternatively, I don't know how many folks _start_ on Identity and then go and find a library that supports TokenCredentials to use - it's probably the other way around right? So maybe best to take this section out?

Reply:
I don't think other languages track what packages use identity and what packages don't use identity. That seems easy to get out of date.

On the problem of "how to identify which packages support identity", what if we open an issue for that?

Packages that use identity could have some label in their package.json, and then we could use a search query to quickly point people to those packages. That sounds like something that could be done nicely, but at the moment is just a vague idea for me at the moment.

Reply:
Created #14684 and removed this section for now